src/main/java/org/example/JAXBMarshaller.java

```diff
@@ -0,0 +1,91 @@
+package org.example;
+
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.Unmarshaller;
+import org.example.model.*;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
+
+public class JAXBMarshaller {
+    public static void main(String[] args) {
+        try {
+            File userXML = new File("src/main/resources/User.xml");
+
+            JAXBContext userContext = JAXBContext.newInstance(User.class);
+            Unmarshaller userUnmarshaller = userContext.createUnmarshaller();
+
+            DocumentBuilder db = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+            Document userDoc = db.parse(userXML);
+            NodeList userNodeList = userDoc.getElementsByTagName("user");
```

Comment:
The requirement for this homework was to parse the xml file using JAXB. Here you initialized  JAXBContext and Unmarshaller but after that you used DOM parser for parsing your xml file. I think that is wrong.

Reply:
Will do. I did this as a way to access the wrapper element before I knew there was an wrapper annotation

Reply:
Ok, I got it. I created a wrapper class just like you mentioned to overcome this issue. By doing so, I feel my code is much more concise. If you feel this works for you, you can keep it like this.